solver/src/liquidity.rs

```diff
@@ -17,7 +17,7 @@ pub mod uniswap;
 #[derive(Clone, AsStaticStr, EnumVariantNames, Debug)]
 pub enum Liquidity {
     Limit(LimitOrder),
-    Amm(AmmOrder),
+    Amm(ConstantProductOrder),
```

Comment:
Should this also be:
```suggestion
    ConstantProduct(ConstantProductOrder),
```